RELEASING.md

```diff
@@ -44,25 +50,24 @@ and assign it to @bencochran or @aquageek.*
     bundle exec pod trunk push Workflow.podspec
     bundle exec pod trunk push WorkflowTesting.podspec
     bundle exec pod trunk push WorkflowUI.podspec
+    bundle exec pod trunk push WorkflowSwiftUI.podspec
     ```
 
-1. Bump the version
+1. Checkout `master` and bump the version (if doing a new release)
    - **Kotlin:** Update the `VERSION_NAME` property in `kotlin/gradle.properties` to the new
      snapshot version, e.g. `VERSION_NAME=0.2.0-SNAPSHOT`.
    - **Swift:** Update `s.version` in `*.podspec` to the new version, e.g. `0.2.0`.
+   ```bash
+   git checkout master
+   sed -i '' -e 's/0.1.0-SNAPSHOT/0.2.0-SNAPSHOT/g' kotlin/gradle.properties 
+   ls *.podspec | xargs sed -i '' -e 's/0.1.0/0.2.0/g'
+   ```
```

Comment:
Using commands to update config manually and then just committing to master blindly makes me nervous, because if someone copy/pastes this command and forgets to change both values, they've suddenly pushed bad code to master without noticing.

Reply:
I added it in because we have like 4 `.podspec` files now :) 

But maybe we can do that on our own 🤷‍♂ 

I'll drop this.